README.md

```diff
@@ -162,6 +162,18 @@ npm run start.desktop
 npm run start.desktop.windows
 ```
 
+#### Develop with livesync
+
+##### prerequisites
+```
+npm install -g electron-prebuilt
```

Comment:
You can remove this part of the instructions since `electron-prebuilt` is part of the seed's main dep's: https://github.com/NathanWalker/angular2-seed-advanced/blob/master/package.json#L66